src/serializer/ResidualFunctions.js

```diff
@@ -186,6 +186,39 @@ export class ResidualFunctions {
       }
     };
 
+    // Emit code for ModifiedBindings for residual functions
```

Comment:
Do you mean "additional" function instead of "residual" function? Also, there are several code parts for modified bindings, can you elaborate what exact code it is trying to emit?